src/Yoti.Auth/CryptoEngine.cs

```diff
@@ -26,20 +26,20 @@ public static AsymmetricCipherKeyPair LoadRsaKey(StreamReader keyStream)
 
         public static byte[] DecipherAes(byte[] key, byte[] iv, byte[] cipherBytes)
         {
-            KeyParameter keyParam = new KeyParameter(key);
-            ParametersWithIV keyParamWithIv = new ParametersWithIV(keyParam, iv);
+            var keyParam = new KeyParameter(key);
```

Comment:
Why use _var_ everywhere if you know what the expected types are?

Reply:
Ok.  The advantage of _var_ seems to be it allows for anonymous types.
https://stackoverflow.com/questions/3425966/what-advantages-does-using-var-have-over-the-explicit-type-in-c

Reply:
I'm not keen on this.  If we don't have a reason _not_ to specify an explicit type, I don't see why we wouldn't want to.  Using _var_ seems to imply that the variable could be something other than _KeyParameter_, but that's not our intention

Reply:
I would agree if it was something like `var i = "42"`, where the compiler just converts it to a string. 

But here, using var when the type is declared on the RHS will always use that type, It's really just a coding style.